.github/workflows/base-ci-goreleaser.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
           platforms: arm64,ppc64le,linux/arm/v7,s390x
 
       - name: Setup wixl # Required to build MSI packages for Windows
-        if: ${{ matrix.GOOS }} == 'windows' && (${{ matrix.GOARCH }} == '386' || ${{ matrix.GOARCH }} == 'amd64')
```

Comment:
Can we split this fix and the check in base-release into a separate PR?

Reply:
sure

Reply:
https://github.com/open-telemetry/opentelemetry-collector-releases/pull/610